packages/data-service/src/data-service.ts

```diff
@@ -1,6 +1,5 @@
 import { callbackify } from 'util';
```

Comment:
Do i understand it correctly that for COMPASS-6620 we will follow up with other PRs?

Reply:
Yes, I'm just doing them in batches to make reviewing easier (and even then managing to mess it up, lol 😅)